style.css

```diff
@@ -0,0 +1,128 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+:root {
+    /***** gradients *****/
+    --scissors-gradient-one: hsl(39, 89%, 49%);
+    --scissors-gradient-two: hsl(40, 84%, 53%);
+    --paper-gradient-one: hsl(230, 89%, 62%);
+    --paper-gradient-two: hsl(230, 89%, 65%);
+    --rock-gradient-one: hsl(349, 71%, 52%);
+    --rock-gradient-two: hsl(349, 70%, 56%);
+    --lizard-gradient-one: hsl(261, 73%, 60%);
+    --lizard-gradient-two: hsl(261, 72%, 63%);
+    --cyan-gradient-one: hsl(189, 59%, 53%);
+    --cyan-gradient-two: hsl(189, 58%, 57%);
+    /***** text *****/
+    --dark-text: hsl(229, 25%, 31%);
+    --score-text: hsl(229, 64%, 46%);
+    --header-outline: hsl(217, 16%, 45%);
+    font-family: 'Barlow Semi Condensed', sans-serif;
```

Comment:
I think it will be best to put this property on the body/html element

Reply:
Fixed.